recipes/pykml/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install 
```

Comment:
You do not need this file since you have the `build: script:` entry in the .yaml file.